.travis

```diff
@@ -0,0 +1,4 @@
+language: scala
+scala:
+   - 2.10.0
```

Comment:
I think we should the latest version here, 2.11.7.

Also, Jenkins uses Java 7 by default which is not supported by Play 2.4. We need to specify JDK8. See here: https://github.com/CSSE497/pathfinder-webserver/blob/dev/.travis.yml